src/main/java/com/fjb/sunrise/controllers/AuthController.java

```diff
@@ -1,10 +1,15 @@
 package com.fjb.sunrise.controllers;
 
 import static com.fjb.sunrise.utils.Constants.ApiConstant.AUTH_REDIRECT_LOGIN;
+import static com.fjb.sunrise.utils.Constants.ApiConstant.AUTH_VERIFICATION;
```

Comment:
Consider removing these constants from the import area and use 'Constants.ApiConstant...' within the code body instead.